native-addons-are-close-to-being-on-par-with-js-modules.md

```diff
@@ -0,0 +1,130 @@
+# Native Addons Are Close To Being On Par With JS Modules
+
+Recent versions of Node.js have introduced features that remove many of the additional maintenance burdens that native addon maintainers have so far experienced compared to maintainers of JavaScript-only packages.
```

Comment:
I would move this to be just before the table, and tweaked to say that the table reflects what was historically the case, with recent versions of Node.js have addressed a number of the issue as discussed later in the article.